<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowVoltimetr</class>
 <widget class="QMainWindow" name="WindowVoltimetr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Источник питания</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/newPrefix/1.bmp</normaloff>:/newPrefix/1.bmp</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Выход</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QFrame" name="frameIMeas">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frameUMeas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Установка</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QFrame" name="frameUSet">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbHolderI">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbHolderU">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbExactly">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>25</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Грубо</string>
     </property>
    </widget>
    <widget class="QFrame" name="frameISet">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>180</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frameHolderU">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frameHolderI">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbStop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>99</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStart">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>99</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEnable">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>99</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Вкл</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>581</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSaveProgram">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>134</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLoadPorgram">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>134</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="aExit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menuMode">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Режим</string>
     </property>
     <addaction name="aManualMode"/>
     <addaction name="aAutoMode"/>
    </widget>
    <addaction name="menuMode"/>
    <addaction name="aSettings"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="aExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="aManualMode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ручной</string>
   </property>
  </action>
  <action name="aAutoMode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Автоматический</string>
   </property>
  </action>
  <action name="aSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="aDeleteRow">
   <property name="text">
    <string>Удалить строку</string>
   </property>
  </action>
  <action name="aInsertRow">
   <property name="text">
    <string>Вставить строку</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
